config/pipelines/pipeline.conf

```diff
@@ -123,6 +123,28 @@ sources=[
     "source": {
       type="com.arpnetworking.metrics.common.sources.CollectdHttpSourceV1"
       name="collectd_http_source"
+    },
+    {
+      type="com.arpnetworking.metrics.common.sources.KafkaSource"
+      name="kafka_source"
+      consumer={
+        type="org.apache.kafka.clients.consumer.Consumer"
+        topics=["topic"]
+        configs={
+          # Set any properties defined at: https://kafka.apache.org/documentation/#consumerconfigs
+          bootstrap.servers="localhost:9092"
+          group.id="group0"
+          client.id="consumer0"
+          key.deserializer="org.apache.kafka.common.serialization.StringDeserializer"
+          value.deserializer="org.apache.kafka.common.serialization.StringDeserializer"
+          auto.offset.reset="earliest"
+        }
+      }
+      parser={
+        type="com.arpnetworking.test.StringParser"
+        name="StringParser"
+      }
+      pollTimeMillis=1000
```

Comment:
We usually express these using ISO8601 durations (e.g. `"PT1S"`). I can point you to an example in the config/code of how to parse these if you like.